tutorials/FITS-tables/FITS_tables.ipynb

```diff
@@ -101,242 +121,388 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "yEuBPS8ycpOB",
+    "outputId": "8bc4c565-e09c-4de0-d99b-1d05fcfb00a3"
+   },
    "outputs": [],
    "source": [
     "hdu_list.info()"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "FCoyC8sIcpOC"
+   },
    "source": [
     "In this case, we're interested in reading EVENTS, which contains information about each X-ray photon that hit the detector."
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "kTHnKQPicpOC"
+   },
    "source": [
     "To find out what information the table contains, let's print the column names."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "CfQyzNPNcpOC",
+    "outputId": "244569b7-9235-43e1-f471-d0789a538b0a"
+   },
    "outputs": [],
    "source": [
     "print(hdu_list[1].columns)"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "YpH2-4qPcpOC"
+   },
    "source": [
     "Now we'll take this data and convert it into an [astropy table](http://docs.astropy.org/en/stable/table/). While it's possible to access FITS tables directly from the ``.data`` attribute, using [Table](http://docs.astropy.org/en/stable/api/astropy.table.Table.html#astropy.table.Table) tends to make a variety of common tasks more convenient."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "id": "n7zMyhRhcpOD"
+   },
    "outputs": [],
    "source": [
     "evt_data = Table(hdu_list[1].data)"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "sEmGEMR1cpOD"
+   },
    "source": [
     "For example, a preview of the table is easily viewed by simply running a cell with the table as the last line:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 1000
+    },
+    "id": "c00cCz9McpOD",
+    "outputId": "86eda857-3fff-4960-f488-477ee510a85c"
+   },
    "outputs": [],
    "source": [
     "evt_data"
```

Comment:
I have used it perhaps once, but I am very unfamiliar with it. I will look it up!

Reply:
Ok, I think I would rather not risk it to add any pprint method here since I looked it up and feel like not very confident about it. I will ask my groupmates to see if they know how to use it.

Reply:
See my suggested changes. We figured out the pprint method and altered the introductory text during our meeting today.